st2common/st2common/exceptions/packs.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 The StackStorm Authors.
+# Copyright 2019 Extreme Networks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from st2common.exceptions import StackStormBaseException
+
+__all__ = [
+    "ResourceDiskFilesRemovalError",
+]
+
+
+class ResourceDiskFilesRemovalError(StackStormBaseException):
+    pass
```

Comment:
Can you move this custom exception to https://github.com/StackStorm/st2/blob/master/st2common/st2common/exceptions/content.py?